<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoForm</class>
 <widget class="QWidget" name="InfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Graphs</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="plot_node" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QCustomPlot" name="plot_alpha" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QCustomPlot" name="plot_beta" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="QCustomPlot" name="plot_eta" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="QCustomPlot" name="plot_tau" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Statistics</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Sum</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Steps</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLCDNumber" name="lcd_steps">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Nodes</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLCDNumber" name="lcd_nodes">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Alphas</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLCDNumber" name="lcd_alphas">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Betas</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLCDNumber" name="lcd_betas">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Etas</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLCDNumber" name="lcd_etas">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Taus</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLCDNumber" name="lcd_taus">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Actual</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="2" column="0">
           <widget class="QLCDNumber" name="lcd_step">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Step</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Nodes</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLCDNumber" name="lcd_node">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Alphas</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLCDNumber" name="lcd_alpha">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Betas</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLCDNumber" name="lcd_beta">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Etas</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLCDNumber" name="lcd_eta">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Taus</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLCDNumber" name="lcd_tau">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="digitCount">
             <number>20</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
